app/application/views.py

```diff
@@ -35,3 +35,33 @@ def view(user_id):
     if not user.application:
         return redirect(404)
     return render_template('application/profile.html', application=user.application)
+
+@login_required
+@application.route('<int:user_id>/change_status_to_complete')
+def change_status_to_complete(user_id):
+    application = Application.query.join(User, User.application_id == Application.id).filter(User.id == user_id).first()
+    if application is None:
+        abort(404)
+
+    checklist_complete = True
+
+    for c in application.user_checklist_items:
+        if c.documents == None:
+            checklist_complete = False
+
+    if checklist_complete:
+        try:
+            application.stage = Stage.COMPLETED_CHECKLIST
+            db.session.commit()
+            flash(
+                'Application stage {} successfully changed to completed checklist.'.format(
+                    application.user), 'application-stage-update-success')
+        except Exception as e:
+            db.session.rollback()
+            flash('Error Occurred. Please try again.', 'application-stage-update-error')
+    else:
+        flash('Checklist not complete. Some documents were not uploaded.', 'application-stage-update-error')
+
+    user = User.query.join(Application, User.application_id == Application.id).filter(User.application_id != None).all()
+    return render_template(
```

Comment:
Just realized that you're querying all users with `user` – be careful because the parameter we use should actually be `applicants` (but shouldn't matter if we just redirect!)